types/node-mailjet/index.d.ts

```diff
@@ -0,0 +1,285 @@
+// Type definitions for node-mailjet 3.3
+// Project: https://github.com/mailjet/mailjet-apiv3-nodejs
+// Definitions by: Nikola Andreev <https://github.com/Nikola-Andreev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export as namespace MailJet;
```

Comment:
I believe this library doesn't expose a UMD module. If that's the case, then this line is incorrect and should be removed.

Reply:
Thanks @armanio123 , i removed the main namespace